integration-tests/load/ccip/ccip_test.go

```diff
@@ -0,0 +1,184 @@
+		if err != nil {
+			lggr.Errorw("Failed to initialize DestinationGun for", "chainSelector", cs, "error", err)
+			t.Fatal(err)
+		}
+
+		otherChains := env.AllChainSelectorsExcluding([]uint64{cs})
+		finalSeqNrCommitChannels[cs] = make(chan finalSeqNrReport)
+		finalSeqNrExecChannels[cs] = make(chan finalSeqNrReport)
+
+		wg.Add(1)
+		go subscribeDeferredCommitEvents(
+			ctx,
+			lggr,
+			state.Chains[cs].OffRamp,
+			otherChains,
+			&block,
+			loki,
+			cs,
+			env.Chains[cs].Client,
+			finalSeqNrCommitChannels[cs],
+			errChan,
+			&wg)
+
+		wg.Add(1)
+		go subscribeExecutionEvents(
+			ctx,
+			lggr,
+			state.Chains[cs].OffRamp,
+			otherChains,
+			&block,
+			loki,
+			cs,
+			env.Chains[cs].Client,
+			finalSeqNrExecChannels[cs],
+			errChan,
+			&wg)
+	}
+
+	loadDuration, err := time.ParseDuration(*userOverrides.LoadDuration)
+	require.NoError(t, err)
+	requestFrequency, err := time.ParseDuration(*userOverrides.RequestFrequency)
+	require.NoError(t, err)
+
+	for _, gun := range gunMap {
+		p.Add(wasp.NewGenerator(&wasp.Config{
+			T:           t,
+			GenName:     "ccipLoad",
+			LoadType:    wasp.RPS,
+			CallTimeout: 5 * time.Second,
+			// 1 request per second for n seconds
+			Schedule: wasp.Plain(1, loadDuration),
+			// limit requests to 1 per duration
+			RateLimitUnitDuration: requestFrequency,
+			// will need to be divided by number of chains
+			// this schedule is per generator
+			// in this example, it would be 1 request per 5seconds per generator (dest chain)
+			// so if there are 3 generators, it would be 3 requests per 5 seconds over the network
+			Gun:        gun,
+			Labels:     CommonTestLabels,
+			LokiConfig: wasp.NewLokiConfig(config.CCIP.Load.LokiEndpoint, nil, nil, nil),
+			// use the same loki client using `NewLokiClient` with the same config for sending events
+		}))
+	}
+
+	_, err = p.Run(true)
+
+	for destSel, gun := range gunMap {
+		for csPair, seqNums := range gun.seqNums {
+			lggr.Debugw("pushing finalized sequence numbers for ",
+				"chainSelector", gun.chainSelector,
+				"sourceChainSelector", csPair.SourceChainSelector,
+				"seqNums", seqNums)
+			finalSeqNrCommitChannels[csPair.DestChainSelector] <- finalSeqNrReport{
+				sourceChainSelector: csPair.SourceChainSelector,
+				expectedSeqNrRange: ccipocr3.SeqNumRange{
+					ccipocr3.SeqNum(seqNums.Start.Load()), ccipocr3.SeqNum(seqNums.End.Load()),
+				},
+			}
+
+			finalSeqNrExecChannels[csPair.DestChainSelector] <- finalSeqNrReport{
+				sourceChainSelector: csPair.SourceChainSelector,
+				expectedSeqNrRange: ccipocr3.SeqNumRange{
+					ccipocr3.SeqNum(seqNums.Start.Load()), ccipocr3.SeqNum(seqNums.End.Load()),
+				},
+			}
+		}
+
+		defer close(finalSeqNrCommitChannels[destSel])
+		defer close(finalSeqNrExecChannels[destSel])
+	}
+
+	wg.Wait()
+	lggr.Infow("finished wait group")
+	for err := range errChan {
+		require.NoError(t, err)
+	}
```

Comment:
Shouldn't this capture some sort of result? How many messages are expected and how will throughput regressions determined?